proto/storage/blob.proto

```diff
@@ -10,10 +10,10 @@ package storage;
 
 // Next Tag: 7
 message Blob {
-    string name                              = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    string name                              = 1 [(gogoproto.moretags) = 'sql:"pk" search:"Blob Name,hidden"'];
```

Comment:
Why is this hidden? I don't think these are exposed externally / they shouldn't be exposed externally to customers so no need to hide